ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/embeddedActivityBlock/embeddedActivityBlock.component.html

```diff
@@ -1,13 +1,23 @@
 <ng-container *ngIf="!activity; else sections">
-  <mat-spinner></mat-spinner>
+    <mat-spinner></mat-spinner>
 </ng-container>
 
 <ng-template #sections>
-  <ddp-activity-section *ngFor="let section of activity.sections; trackBy: getSectionId"
-                        [section]="section"
-                        [studyGuid]="studyGuid"
-                        [activityGuid]="instance.instanceGuid"
-                        [readonly]="readonly"
-                        [validationRequested]="validationRequested">
-  </ddp-activity-section>
+    <div class="instance__divider"></div>
```

Comment:
**minor** can material divider be used here? https://material.angular.io/components/divider/overview

Reply:
The idea was to have something as a divider (hidden by default) between embedded blocks and be able to customize it (by css) in every study. 
The easiest stuff is a `div` there. 
Can you provide some benefits if we use material divider ?

Reply:
the benefit which I can see here: no additional css :)